docker/drone/src/main/java/org/arquillian/cube/docker/drone/event/AfterConversion.java

```diff
@@ -0,0 +1,11 @@
+package org.arquillian.cube.docker.drone.event;
+
+import org.jboss.arquillian.core.api.Event;
+
+public class AfterConversion implements Event {
```

Comment:
You need to implement `org.jboss.arquillian.core.spi.event.Event` not `org.jboss.arquillian.core.api.Event`